Sources/SwiftKuery/Column.swift

```diff
@@ -84,8 +114,23 @@ public class Column: Field, IndexColumn {
         self.collate = collate
     }
     
-    /// Build the column using `QueryBuilder`.
-    ///
+    //MARK: Column Decription Functions
+    
+    /**
+     Function to build a String representation for referencing a `Column` instance.
+     A `QueryBuilder` is used handle variances between the various database engines and produce a correct SQL description.
+     This function is required to obey the `Buildable` protocol.
+     ### Usage Example: ###
+     In this example, `QueryBuilder` and a `PersonTable` (as defined at the top of this class) instances are initialized.
+     The build function is then called to produce a String description and the results are printed.
+     ```swift
+     let queryBuilder = QueryBuilder()
+     let personTable = PersonTable()  // PersonTable() is a previously defined `Table` class
+     let description = try personTable.name.build(queryBuilder: queryBuilder)
+     print(description)
+     // Prints personTable.name
+     ```
+     */
```

Comment:
I'm not seeing API doc for any of the Parameter, Returns or Throws for any of the functions within these updated Classes within Swift-Kuery???